### PR TITLE
LPS-85300 Ignore Separator Fields

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

File: modules/apps/headless-apio/structured-content/structured-content-apio-api/src/main/java/com/liferay/structured/content/apio/architect/resource/StructuredContentField.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+package com.liferay.structured.content.apio.architect.resource;
+
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Holds information about a structured content field.
+ *
+ * @author Cristina González
```
Comment (nhpatt) [inline modules/apps/headless-apio/structured-content/structured-content-apio-api/src/main/java/com/liferay/structured/content/apio/architect/resource/StructuredContentField.java]: @review :(

File: modules/apps/headless-apio/structured-content/structured-content-apio-api/src/main/java/com/liferay/structured/content/apio/architect/resource/StructuredContentField.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	 *
+	 * @return the input control.
+	 *
+	 * @review
+	 */
+	public String getInputControl();
+
+	/**
+	 * Returns the label of the structured content given a locale.
+	 * @param  locale the locale.*
```
Comment (nhpatt) [inline modules/apps/headless-apio/structured-content/structured-content-apio-api/src/main/java/com/liferay/structured/content/apio/architect/resource/StructuredContentField.java]: small typo

**Comment by ruben-pulido** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ruben-pulido** [comment]
Thanks @cgoncas! 
I've added @nhpatt's feedback and a couple of minor changes and have submitted it to Brian:
https://github.com/brianchandotcom/liferay-portal/pull/64976

**Pull request closed without merging**